pages/10.contacts/03.manage-segments/docs.en.md

```diff
@@ -143,6 +143,14 @@ manually added will remain part of the list regardless of filters.
 In addition to segments you can also manually add any contact to a list by
 clicking the Preferences button at the segments tab, use the dropdown to select a segment and add the contact to it or click on the x next to a segment in the input field to remove the contact.
 
+## Add categories to segment
+
+Apart from organizing various Mautic elements, categories are used to organize contact. To add categories to segment, create a new [category](../categories#creating-managing-categories) of type `segment`. When you add a new segment, you can now sort and select from the categories you previously created.
+
+![Screenshot showing segment categories](add-category-segment.jpg)
+
+To check if your segment is linked to the categories you created, create a new uncategorized segment and click on the `Segment Category Filter` in the toolbar and select the category which you linked to your segment. Only that segment will be listed the uncategorized segment which is not linked to any category wont be listed.
```

Comment:
```suggestion
To determine whether your segment is linked to the categories you created, first create a new segment without any category assigned. From the Segments screen, click in the `Segment Category Filter` in the toolbar. Choose the category linked to your segment. Only the segment associated with that category is displayed, the other segments are not listed.
```

Reply:
@RCheesley I was trying a screenshot that might show these steps, what do you think